src/components/Calculator/Digit.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+// 버튼을 눌르면 Total Component가 업데이트
+type DigitProps = {
+    handleClick: React.MouseEventHandler<HTMLButtonElement>;
+};
+
+export const Digit = ({ handleClick }: DigitProps) => {
+    const digitButtons=[0,1,2,3,4,5,6,7,8,9].reverse()
```

Comment:
```suggestion
    const digits = [9, 8, 7, 6, 5, 4, 3, 2, 1, 0];
```
단순한 배열이라 button 네이밍은 안들어가도 될 것 같아요. 👀